cmd/osbuild-composer/composer.go

```diff
@@ -90,6 +97,9 @@ func (c *Composer) InitWeldr(repoPaths []string, weldrListener net.Listener) err
 	if err != nil {
 		return fmt.Errorf("error loading repository definitions: %v", err)
 	}
+	if fallback {
+		rr.AddRepos(fallback_name, fallback_repos)
+	}
```

Comment:
I'm wondering whether these fallback repositories could be added already when constructing the `RepoRegistry`, to not to make a special case out of them. But I don't have a specific idea how to do it... :smile: